### DISCUSSION TITLE
Sparse Bilatteral filtering

### DISCUSSION THREAD

**Discussion by thygate** [graemeniedermayer/stable-diffusion-webui-normalmap-script#4] [Ideas]
In preparation of integrating the 3D-Photo-Inpainting paper into the depth extension, I've copied over the sparse_bilateral_filtering function from said repository. But in my (brief) testing I cannot notice any noticeable changes in its output compared to the original depth map.

As you are more informed in the subject matter, and suggested you wanted to implement it, perhaps you would like to take a look at it and determine if it's useful ? It's in the current version of the depthmap extension, but commented out ..

Perhaps I have not been able to generate images containing the type of noise it can smooth ..

**Comment by thygate** [comment]
I was not feeding it the rgb map correctly, is why i initially couldn't see any difference. The 3D inpainting requires it to function. It's implemented correctly now.

**Reply by graemeniedermayer** [reply]
Sorry for the late reply! I'm happy you got it working!
